File: RemoveVowelsFromString.py
# Remove Vowels From String
# - Given a string `s`, remove all vowels (a, e, i, o, u, both cases).
# - Constraints:
#   - 1 <= len(s) <= 10^4

class Solution:
    def removeVowels(self, s: str) -> str:
        vowels = "aeiouAEIOU"
        return "".join(c for c in s if c not in vowels)

File: RemoveVowelsFromString.cs
// Remove Vowels From String
// - Given a string `s`, remove all vowels (a, e, i, o, u, both cases).

using System;
using System.Collections.Generic;
using System.Linq;

class Solution
{
    public string RemoveVowels(string s)
    {
        string vowels = "aeiouAEIOU";
        return new string(s.Where(c => !vowels.Contains(c)).ToArray());
    }
}
